tests/test_messages_config/test_messages_config.py

```diff
@@ -2,40 +2,85 @@
 import sys
 
 
-def test_modified_message():
-    """Test the presence of modified custom message."""
+def test_valid_message_config_and_pyta_overwrite():
+    """Test the error messages given a valid messages-config-path with use-pyta-error-messages as True"""
     output = subprocess.run(
         [
             sys.executable,
             "-m",
             "python_ta",
             "--config",
-            "tests/test_messages_config/test.pylintrc",
-            "examples/pylint/e0111_bad_reversed_sequence.py",
+            "tests/test_messages_config/test1.pylintrc",
```

Comment:
The test cases are all good, but you should pick more meaningful names for each configuration file. You can name them whatever you want, so pick names that suggest what test case they're being used for.